app/server/api-generator/src/test/resources/soap/StockQuote.wsdl

```diff
@@ -26,7 +26,7 @@
 
     <message name="GetTradePriceInput">
         <part name="tickerSymbol" type="xsd:string"/>
-        <part name="time" type="xsd:timeInstant"/>
```

Comment:
Does the WS-I Basic Profile 2.0 say anything about this ambiguity? Would be nice to point to some standards doc to say why we support it this way.

Reply:
It's a change introduced in SOAP 1.2: https://www.w3.org/2000/xp/Group/1/10/11/soap12-part1.html#schemachanges, BP 2.0 is based on SOAP 1.2: http://ws-i.org/profiles/BasicProfile-2.0-WGD.html#bp11compatibility